components/ContentPage.tsx

```diff
@@ -11,12 +12,15 @@ export const pageComponents = {
 }
 
 interface ContentPageProps {
-  page: Page
+  page?: Page
 }
 
 export const ContentPage = ({ page }: ContentPageProps) => {
   return (
     <div className="w-full sm:w-1/3">
+      <Head>
+        <title>{page?.title} - The Atlas of Ownership</title>
```

Comment:
For "About" this is curiously reading from the `h1` & showing `Other forms of ownership are possible - The Atlas of Ownership` which doesn't feel perfectly ideal but isn't a dealbreaker! 

Is there an easy way to set a different title that reflects route versus displayed h1 in edge cases like this?

Reply:
Good catch - added a commit to address this.